app/src/main/java/org/mozilla/fenix/tabstray/syncedtabs/SyncedTabs.kt

```diff
@@ -310,32 +351,47 @@ private fun SyncedTabsListPreview() {
 /**
  * Converts a list of [SyncedDeviceTabs] into a list of [SyncedTabsListItem].
  */
-fun List<SyncedDeviceTabs>.toComposeList() = asSequence().flatMap { (device, tabs) ->
-    // Transform to sticky headers data here https://github.com/mozilla-mobile/fenix/issues/19942
-    val deviceTabs = if (tabs.isEmpty()) {
-        sequenceOf(SyncedTabsListItem.NoTabs)
+fun List<SyncedDeviceTabs>.toComposeList(): List<SyncedTabsListItem> = asSequence().flatMap { (device, tabs) ->
```

Comment:
I tempted to say move this to ext/SyncedDeviceTabs

Reply:
Makes sense to me